### DISCUSSION TITLE
Recommended branch to run on?

### DISCUSSION THREAD

**Discussion by gx1400** [magicbug/Cloudlog#2099] [General]
Which branch or release is recommended to be tracked for a deployed instance?

Master? Or the newest release? 

Do the releases have a notification when a new release is pushed or do you need to monitor/alert github ?

**Comment by AndreasK79** [comment]
Well, the master is the most stable release branch. The dev branch might be unstable, or some things will not work. It's really up to you.

No notification exists for a new release. If you are on a Linux system, you can set up a cron job to run the update script.

**Reply by gx1400** [reply]
Understand all of the above. Thanks for the confirmation.

Maybe that's an opportunity for a feature improvement, it would be nice to run on Release branches then get a dismiss-able banner when a new release has been issued
